### PR TITLE
CrowdStrike Adding Pagination

### PR DESCRIPTION
**Pull request by warpo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:  [Marketplace Contribution] CrowdStrike Falcon - Content Pack Update #22934 review

## Description
Updates based on the review to PR 22934

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by warpo** [comment]
I'm trying to commit new changes to an existing PR #22934 , but I was having issues in GitHub doing so. Hopefully this will suffice to cover the reviewers comments. @JasBeilin

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_0.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CrowdStrike Falcon
+- %%UPDATE_RN%%
```
Comment (JasBeilin) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_0.md:4]:
```suggestion
- Added support for the *after* argument in the ***cs-falcon-search-custom-iocs*** command.```

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Great job, 
I will close the old PR for you. As this is not contribution through the Marketplace, I will need you to fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA).
I would also appreciate if you can elaborate some more on the committing issue as it might be affecting other contributions as well.

Please note that the build is failing on unit tests and a few simple validations.
Let me know if you need help in fixing those so we can move forward. 
Can we schedule a short demo session where you show me this code addition working as expected?

**Comment by JasBeilin** [comment]
Hi @warpo, I am unavailable today, but I sent an invite for next week. Let me know if you need me to reschedule it. Are you part of DFIR community in slack? This is the best way to communicate with us outside of the PR.

**Comment by warpo** [comment]
> Hi @warpo, I am unavailable today, but I sent an invite for next week. Let me know if you need me to reschedule it. Are you part of DFIR community in slack? This is the best way to communicate with us outside of the PR.

Hello @JasBeilin. I did receive the meeting invite and will be joining. See you then.

**Comment by warpo** [comment]
@JasBeilin I've completed the changes that we discussed. I had to update a couple of the tests to accommodate for the new output type from the search command. We can now see both the IOC list and the Pagination Token as Human Readable output. Please have a look and we can discuss tomorrow during our review.

**Review by JasBeilin — APPROVED** [review]
Great job! Thank you so much for this contribution.

**Pull request merged**